docs/reference/logging/overview.md

```diff
@@ -107,6 +111,14 @@ the following named facilities:
 * `local6`
 * `local7`
 
+`syslog-tls-ca-cert` specifies the absolute path in the host to the Trust certs signed by the CA. This option is ignored if the address protocol is not `tcp+tls`.
```

Comment:
oh, and absolute path in the host -> on the host? (or simply "the absolute path")